sdk/ml/azure-ai-ml/tests/component/e2etests/test_component.py

```diff
@@ -229,7 +231,7 @@ def test_automl_component(self, client: MLClient, registry_client: MLClient, ran
         )
 
         assert_component_basic_workflow(
-            client=registry_client,
+            client=data_asset_registry_client,
```

Comment:
why is this called **data_asset** _registry_client?

Reply:
I'll revert this change